frontend/packages/ux-editor/src/testing/schemas/json/component/Input.schema.v1.json

```diff
@@ -170,7 +180,6 @@
       "properties": {
         "currency": {
           "title": "Language-sensitive currency formatting",
-          "type": "string",
```

Comment:
Removing this makes the currency dropdown disappear 🤔 Is this supposed to happen?

Without `"type": "string"`:
![image](https://github.com/Altinn/altinn-studio/assets/148075168/55874cf6-f459-4efd-b776-b50fae60fbe9)

With `"type": "string"`:
![image](https://github.com/Altinn/altinn-studio/assets/148075168/1f259f72-52d3-4026-afd5-6b49adee6565)

Reply:
Hm, not a lucky outcome... But this is the case for many of the other properties on the formatting for Input component unfortunately. So yes, we could keep the original schema and also modify the other properties to include the `type: string` to make also them visible and editable. But since we rely on generating changes in component schemas from team apps through the script, we will always get diffs where these studio-modified props are deleted 🤔

I think a better solution would be to create a new issue where we can identify these properties in a different way. Maybe to use the enum-property directly 🫣

Reply:
Yes, that sounds like a good solution😊
But should we add `"type": "string"` to these properties until that is fixed?

Reply:
Just to be clear; I have not removed this field, it is removed due to the script that uses the schemas from cdn which is updated by the apps team to not include the `type: string`. A thing we can/should do anyway is to ask them why they are removing it from many properties 🫣 I can ask them on Slack!

Reply:
After clearification with team Apps it is by design that this property is removed so we simply need to adjust to that and render these configs based on the `enum` field. I have created an issue for it 🤗 
https://github.com/Altinn/altinn-studio/issues/13127

Reply:
Good 😄